docker/aladdin-demo.Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM alpine:latest
+
+# install python and pip with apk package manager
+RUN apk -Uuv add python py-pip
+
+# copies requirements.txt to the docker container
+ADD requirements.txt requirements.txt
```

Comment:
I think it's best to pu this after the apk add command, and then do the pip install after that. Although it creates more layers, it won't reinstall the packages everytime you change requirements.txt (because of how docker layer caching works). Let me know if that doesn't make sense.